presto-main/src/main/java/io/prestosql/sql/planner/planprinter/HashCollisionPlanNodeStats.java

```diff
@@ -83,7 +83,7 @@ private static double computedWeightedStdDev(double sumSquared, double sum, doub
     @Override
     public PlanNodeStats mergeWith(PlanNodeStats other)
     {
-        checkArgument(other instanceof HashCollisionPlanNodeStats, "other is not an instanceof HashCollisionPlanNodeStats");
+        checkArgument(other instanceof HashCollisionPlanNodeStats, "other is not an instanceof HashCollisionPlanNodeStats: %s", other);
```

Comment:
I'm pretty sure the error would be about plain `PlanNodeStats`

Reply:
Right. See follow up https://github.com/prestosql/presto/pull/6091 too.